lib/App/DuckPAN.pm

```diff
@@ -275,6 +275,17 @@ sub _build_ddg {
 	App::DuckPAN::DDG->new( app => shift );
 }
 
+has fathead => (
```

Comment:
@moollaza Ah, so it is utility functions for FatHeads rather than information on the particular FatHead being run on?